testing/jormungandr-integration-tests/src/common/jormungandr/process.rs

```diff
@@ -209,10 +285,13 @@ impl SyncNode for JormungandrProcess {
     }
 
     fn get_lines_with_error_and_invalid(&self) -> Vec<String> {
-        self.logger.get_lines_with_error_and_invalid().collect()
+        self.logger
+            .get_lines_with_level(LogLevel::ERROR)
+            .map(|x| x.to_string())
+            .collect()
     }
 
     fn is_running(&self) -> bool {
-        todo!()
+        matches!(self.status(&StartupVerificationMode::Log), Status::Running)
     }
```

Comment:
This is kinda expensive to re-parse all logs in a method simply named `is_running`, but maybe it would not have much impact on tests.

Reply:
they are not actually parsed again, it goes through already parsed lines